src/main/java/com/ft/methodearticlemapper/transformation/RecommendedXMLEventHandler.java

```diff
@@ -0,0 +1,119 @@
+package com.ft.methodearticlemapper.transformation;
+
+import com.ft.bodyprocessing.BodyProcessingContext;
+import com.ft.bodyprocessing.BodyProcessingException;
+import com.ft.bodyprocessing.writer.BodyWriter;
+import com.ft.bodyprocessing.xml.eventhandlers.BaseXMLEventHandler;
+import org.apache.commons.lang.StringUtils;
+
+import javax.xml.stream.XMLEventReader;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.events.StartElement;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RecommendedXMLEventHandler extends BaseXMLEventHandler {
+
+    private static final String RECOMMENDED_TAG = "recommended";
+    private static final String RECOMMENDED_TITLE_TAG = "recommended-title";
+    private static final String INTRO_TAG = "p";
+    private static final String LIST_TAG = "ul";
+    private static final String LIST_ITEM_TAG = "li";
+    private static final String ANCHOR_TAG = "a";
+    private static final String TYPE_ATTRIBUTE = "type";
+    private static final String HREF_ATTRIBUTE = "href";
+    private static final String URL_ATTRIBUTE = "url";
+    private static final String TYPE_VALUE = "http://www.ft.com/ontology/content/Article";
+    private static final String BASE_URL = "http://api.ft.com/content/";
+    private static final String REGEX = "(^\\/[A-Za-z0-9/.?]*=)([a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12})$";
+
+    private Pattern pattern;
+    private RecommendedXMLParser recommendedXMLParser;
+
+    public RecommendedXMLEventHandler(RecommendedXMLParser parser) {
+        this.recommendedXMLParser = parser;
+        this.pattern = Pattern.compile(REGEX);
+    }
+
+    @Override
+    public void handleStartElementEvent(StartElement startElement,
+                                        XMLEventReader xmlEventReader,
+                                        BodyWriter eventWriter,
+                                        BodyProcessingContext bodyProcessingContext) throws XMLStreamException {
+        if (isElementOfCorrectType(startElement)) {
```

Comment:
I saw this check being added in some other Event Handlers: PullQuoteEventHandler, PromoBoxEventHandler, DataTableXMLEventHandler. It may be redundant but I'm not sure.